tests/foreman/ui/test_navigation.py

```diff
@@ -0,0 +1,115 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Login UI
+
+@Requirement: Navigation
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import skip_if_bug_open, tier1
+from robottelo.test import UITestCase
+from robottelo.ui.locators import common_locators, menu_locators
+from robottelo.ui.session import Session
+
+
+class NavigationTestCase(UITestCase):
+    """Implements the navigation tests from UI"""
+
+    def page_objects(self):
+        return {
+            'Activation Key': self.activationkey,
```

Comment:
@oshtaier there's no real need in instance, as attributes which are referenced have constant names, `getattr()` could be used to access them in this case.